solidity/dashboard/src/components/CustomNavLink.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import { NavLink } from "react-router-dom"
+import useWalletAddressFromUrl from "../hooks/useWalletAddressFromUrl"
+
+const CustomNavLink = ({ to, ...props }) => {
```

Comment:
I think we can skip `Custom` and just `NavLink`, `Redirect` etc. IMO w/o `Custom` it will be clear that this is the right component to use in the entire app.